sample-apps/radiology/lib/transforms/transforms.py

```diff
@@ -486,3 +488,38 @@ def __call__(self, data: Mapping[Hashable, np.ndarray]) -> Dict[Hashable, np.nda
                 d[key] = tmp_image
 
         return d
+
+
+class NormalizeLabelsInDatasetd(MapTransform):
+    def __init__(self, keys: KeysCollection, label_names=None, allow_missing_keys: bool = False):
+        """
+        Normalize label values according to label names dictionary
+
+        Args:
+            keys: The ``keys`` parameter will be used to get and set the actual data item to transform
+            label_names: all label names
+        """
+        super().__init__(keys, allow_missing_keys)
+
+        self.label_names = label_names
+
+    def __call__(self, data: Mapping[Hashable, np.ndarray]) -> Dict[Hashable, np.ndarray]:
+        d: Dict = dict(data)
+        for key in self.key_iterator(d):
+            if key == "label":
```

Comment:
can we remove this check.. it's redundant..

Reply:
Yes, thanks!